core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1904,15 +1904,6 @@ public MetricKey build() {
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(false)
           .build();
-  public static final MetricKey CLIENT_TOTAL_RPC_CLIENTS =
-      new Builder("Client.TotalRPCClients")
-          .setDescription("The total number of RPC clients exist that is using to "
-              + "or can be used to connect to master or worker for operations. "
-              + "The sum of the sizes of FileSystemMasterClientPool, "
-              + "BlockMasterClientPool, and BlockWorkerClientPool.")
-          .setMetricType(MetricType.COUNTER)
-          .setIsClusterAggregated(false)
-          .build();
```

Comment:
This one can now be summed from the 3 new metrics. If we do need one syntactic sugar gauge, we can sum the other 3 counters in this gauge, instead of leaving references to peek the pool sizes at all.